infrastructure/sprod.tfvars

```diff
@@ -16,6 +16,6 @@ http_client_max_client_per_route = 40
 #http_client_validate_after_inactivity = 0
 
 definition_cache_latest_version_ttl_sec = 5
+definition_cache_jurisdiction_ttl_sec = 5
```

Comment:
Both of `definition_cache_latest_version_ttl_sec` and `definition_cache_jurisdiction_ttl_sec` must match the values on Prod. Same for AAT